### ISSUE TITLE
Building Locally

### ISSUE THREAD

**Issue by Awesome-XV** [mohsenemx/FXclient#2]
![image](https://github.com/mohsenemx/FXclient/assets/147341789/b3a60130-a724-41cf-a763-042d72d8ef5c)
While building locally this happens to me

**Comment by peshomir** [comment]
Yeah well 3 days ago the creator made an update to the game that broke some of the functionality, lately I haven't had much time to update the client anyways, but I hope I'll be able to update it soon

**Comment by Awesome-XV** [comment]
Alright

**Comment by peshomir** [comment]
The client has been updated
(see https://github.com/mohsenemx/FXclient/commit/f4e8deb)
